analysis/contract/to/analysis_to.py

```diff
@@ -26,6 +27,8 @@ class AnalysisTO(BaseTO):
     startDate: date | None = None,
     creator: Optional[str] = None,
     locations: Optional[str] = None
+    analysisSteps: Optional[str] = None
```

Comment:
This can be types has Optional[list[AnalysisStepTO]]
Also localtions Optional[list[AdministrativeDivisionTO]]